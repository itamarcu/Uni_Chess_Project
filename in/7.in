game_mode 2
start
move <2,E> to <4,E>
move <7,E> to <6,E>
move <2,D> to <4,D>
move <7,D> to <5,D>
move <1,B> to <2,D>
move <7,C> to <5,C>
move <4,E> to <5,D>
move <6,E> to <5,D>
move <4,D> to <5,C>
move <8,F> to <5,C>
move <1,G> to <2,E>
move <8,D> to <6,B>
move <2,A> to <4,A>
move <5,C> to <2,F>
move <1,E> to <2,F>
quit
