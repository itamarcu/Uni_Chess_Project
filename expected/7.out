 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ m m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m _ _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M _ _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ _ M _ _ _ _ |
4| _ _ _ m m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m _ _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M _ _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ _ M _ _ _ _ |
4| _ _ _ m m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M _ _ _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ M M _ _ _ _ |
4| _ _ _ m m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M _ _ _ M M M |
6| _ _ _ _ M _ _ _ |
5| _ _ M m _ _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M _ _ _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ M M _ _ _ _ |
4| _ _ _ m _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M _ _ _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ m M _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K _ N R |
7| M M _ _ _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ B M _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n _ m m m |
1| r _ b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K _ N R |
7| M M _ _ _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ B M _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n n m m m |
1| r _ b q k b _ r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B _ K _ N R |
7| M M _ _ _ M M M |
6| _ Q _ _ _ _ _ _ |
5| _ _ B M _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m n n m m m |
1| r _ b q k b _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B _ K _ N R |
7| M M _ _ _ M M M |
6| _ Q _ _ _ _ _ _ |
5| _ _ B M _ _ _ _ |
4| m _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| _ m m n n m m m |
1| r _ b q k b _ r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Checkmate! black player wins the game
